app/build.gradle

```diff
@@ -65,6 +65,7 @@ if (project.hasProperty('sign')) {
 dependencies {
```

Comment:
The first time I started the app with these changes, it only showed the splashscreen and didn't continue further. I sadly couldn't reproduce it afterwards. Do you have an idea what could've caused this?

Reply:
Are you using an emulator?

Reply:
Yes.

Reply:
Maybe that's the problem. This library has some problems being launched on an emulator from Android Studio.
If you close the app and start it from the UI on the emulator again it should work (if that's really the problem).

Reply:
Yeah, probably a good enough explanation for now.

Reply:
Have you tested it on a physical device as well?
There you should not have that problem.

Reply:
No, haven't tested it on a physical device yet. But I couldn't reproduce the problem afterward, so it was hopefully something cached related from the emulator.

Reply:
I've tested it on two Emulators now (APIs 25 and 33 as well). Everything worked fine, on one device it took a second or two to do something because it had just booted up, but both showed me the login page.

Reply:
I've tested it on physical devices with Android 10 & 13, works fine.